internal/controlplane/server.go

```diff
@@ -352,8 +354,20 @@ func (s *Server) StartHTTPServer(ctx context.Context) error {
 	// This already has _some_ middleware due to the GRPC handling
 	mux.Handle("/", withMaxSizeMiddleware(s.withCORSMiddleware(gwmux)))
 
+	// Register the webhook handlers
+	// Note: The GitHub webhook handler is not registered here.
+	for classpath, handler := range s.providerManager.IterateWebhookHandlers() {
```

Comment:
do we not register the GH handler here because we have a single handler for the app and the oauth providers?

Reply:
ah, no, it's because this adds the API and not the implementation yet, sorry for the noise